front/src/app/core/services/programmer/programmer.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { ICueble } from '../../interfaces/ICueble';
+import { IChannel } from '../../interfaces/IChannel';
+import { ICue } from '../../interfaces/ICue';
+import { GeneralCue } from '../../library/GeneralCue';
+import { ValueCueble } from '../../library/ValueCueble';
+import { IFixture } from '../../interfaces/IFixture';
+
+@Injectable()
+export class ProgrammerService {
+    cuebles: ICueble[];
+    constructor(
```

Comment:
Just for clarification: This will be filled later (for injection etc.), right? Otherwise, this empty constructor would be kind of useless...